cql-ast/src/main/scala/troy/cql/ast/dml/Condition.scala

```diff
@@ -3,3 +3,7 @@ package troy.cql.ast.dml
 import troy.cql.ast.Term
 
 final case class Condition(simpleSelection: SimpleSelection, operator: Operator, term: Term)
+
+sealed trait IfCondition
+final case class SimpleIfCondition(conditions: Seq[Condition]) extends IfCondition
+final case class Exist(value: Boolean) extends IfCondition
```

Comment:
Can you rename this class to `IfExists`, because just `Exists` is too generic, especially that this class is not nested under any objects.